src/modules/sample/SampleAddComponent.jsx

```diff
@@ -75,42 +124,50 @@ const FieldReadOnly = ({field, label, defaultValue, inputProps}: FieldReadOnlyPr
   );
 };
 
-const submitSample = (appSession, form, addSample) => {
+const submitSample = (appSession: AppSession, form: FormData, addSample: Function) => {
   const token = appSession.accessToken;
   const museumId = appSession.museumId;
-  const myReduce = (frm) => Object.keys(frm).reduce((akk: any, key: string) => ({...akk, [key]: frm[key].value}), {});
-  const data = myReduce(form);
+  const myReduce = (frm: FormData) => Object.keys(frm).reduce((akk: any, key: string) => ({...akk, [key]: frm[key].value}), {});
+  const reducePersons = (p: any) => p && p.reduce((akk: any, v: Person) => {
```

Comment:
akk is 
{
  createdBy?: string,
  createdDate?: string,
  responsible?: string
}